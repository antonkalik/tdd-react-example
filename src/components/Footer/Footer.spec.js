import { render, screen } from "@testing-library/react";
import { Footer } from "./index";

describe("Footer", () => {
  beforeEach(() => {
    render(<Footer />);
  });

  it("should render footer", () => {
    expect(screen.getByTestId("footer")).toMatchSnapshot();
  });

  it("should have description", () => {
    const description = screen.getByTestId("footer");
    expect(description).toHaveTextContent("footer description");
  });
});
