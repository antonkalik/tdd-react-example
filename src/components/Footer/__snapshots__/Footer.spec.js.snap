// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render footer 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100px;
  background-color: #777;
}

<footer
  class="c0"
  data-testid="footer"
>
  <p>
    footer description
  </p>
  <p>
    Total Cards: 
    9
  </p>
</footer>
`;
